### DISCUSSION TITLE
"Set up billing to run this model."

### DISCUSSION THREAD

**Discussion by saraivasamanta3** [TencentARC/GFPGAN#488] [General]
I was using gfpgan on replicate.com, now this message appears:
![image](https://github.com/TencentARC/GFPGAN/assets/154033878/1d6c548a-6383-45df-8d21-b66e279890d1)

Is the demo now paid?

**Comment by OmlineEditor** [comment]
it's all free if you log in through github, but if you generate a lot of photos then you need to pay
